app/context-map/workload.js

```diff
@@ -0,0 +1,20 @@
+const Workload = require('../points/domain/workload')
```

Comment:
This should take a workload object from the staging context i.e. `om-workload` then a validation check should be performed to ensure that it is of the correct type.